.github/workflows/e2e.yaml

```diff
@@ -61,8 +64,22 @@ jobs:
           helm repo add newrelic https://helm-charts.newrelic.com
           helm repo add kube-state-metrics https://kubernetes.github.io/kube-state-metrics
           helm repo update
+      - name: Select metrics exception file
+        id: exceptions-version
+        run: |
+          MINOR=$(echo "${{ matrix.k8sVersion }}"|sed -e 's_v\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\)_\2_')
+
+          if [ "$MINOR" -le 22 ]; then
+            echo "::set-output name=exceptions::1_22"
+          elif [ "$MINOR" -eq 23 ]; then
+            echo "::set-output name=exceptions::1_23"
+          elif [ "$MINOR" -ge 24 ]; then 
+            echo "::set-output name=exceptions::1_24"
+          fi
       - name: Run e2e-test
-        uses: newrelic/newrelic-integration-e2e-action@v1
+        uses: newrelic/newrelic-integration-e2e-action@add-metric-except-source
+        env:
+          EXCEPTIONS_SOURCE_FILE: ${{ steps.exceptions-version.outputs.exceptions }}-exceptions.yml
```

Comment:
Can we add a comment in the spec file saying that we are including also this?

Reply:
done added a comment on the spec file